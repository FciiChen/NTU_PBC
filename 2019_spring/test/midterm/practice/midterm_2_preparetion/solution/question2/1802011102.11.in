aQHy8P, 7lreh8ssip,8XPn6,8L5Clsga,caa
G3r, ruK,kWeKDCFy,D4LB,ZLmUm2
kaV6, 59I,UN0XY,1sqOR,MfDQFf2
TXl, 930V9CD,XNtJ4OFsYB,xTDdFqVFbI,0ER
tLeidMbD, wRmVRBnfev,0qad8s,EcRKj5kXei,1awgRhwa
YNCPACx, 3Xm,9Wzkz2fR,RtFb,q6L2Hngf
7DidBSW8, RIB3zf,s6LJnT,DnvaJK6d8,5kK0Jw3
BREAK
YNCPACx ,  bvdK6aqB4
tLeidMbD ,  vvwQdH3b
kaV6 ,  0Tyna0
TXl ,  ilKdNI
BREAK
