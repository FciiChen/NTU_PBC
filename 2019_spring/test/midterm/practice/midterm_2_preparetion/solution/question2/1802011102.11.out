aQHy8P     7lreh8ssip 8XPn6      8L5Clsga   caa        NULL      
G3r        ruK        kWeKDCFy   D4LB       ZLmUm2     NULL      
kaV6       59I        UN0XY      1sqOR      MfDQFf2    0Tyna0    
TXl        930V9CD    XNtJ4OFsYB xTDdFqVFbI 0ER        ilKdNI    
tLeidMbD   wRmVRBnfev 0qad8s     EcRKj5kXei 1awgRhwa   vvwQdH3b  
YNCPACx    3Xm        9Wzkz2fR   RtFb       q6L2Hngf   bvdK6aqB4 
7DidBSW8   RIB3zf     s6LJnT     DnvaJK6d8  5kK0Jw3    NULL      
